### PR TITLE
docs(aws-bedrock): AWS bedrock documentation 

### PR DESCRIPTION
**Pull request by mathias-vandaele**
add complete documentation on how to use the new aws bedrock connector

Linked Issue: https://github.com/camunda/team-connectors/issues/842

Will be release on the next alpha, which should be 8.6.0-alpha4

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
Hi @mathias-vandaele 😄 Is this linked to a particular issue, and set for a particular alpha or release?

**Comment by mathias-vandaele** [comment]
> Hi @mathias-vandaele 😄 Is this linked to a particular issue, and set for a particular alpha or release?

Hi @christinaausley This should be release on the next alpha, which should be 8.6.0-alpha4

**Comment by mathias-vandaele** [comment]
Hey @christinaausley I am ready for a technical and engineering review, as It should now be in the last release

File: docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+### Converse
+
+This action is meant to start or continue a conversation with a model.
+
+A model id has to be specified, you can find all the available one for amazon
+bedrock [here](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html)
+Be sure the model is available on your region, that your model can invoke the `Converse` action, and you are using a user with adequate rights.
+
+`New Message` is either the first message (to start a conversation) or is the next message from an already started conversation
+`Messages History` is the history of the conversation that should always be passed. If not set, this will be a new conversation
```
Comment (sbuettner) [inline docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md:119]: Isnt this called `Message History`?

File: docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables.
+
+The response contains two elements:
+
+`messagesHistory` is the full history of previous message, from user and assistant, including the latest message written by the assistant
```
Comment (sbuettner) [inline docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md:126]:
```suggestion
`messageHistory` is the full history of previous message, from user and assistant, including the latest message written by the assistant
```

**Review by sbuettner — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables.
+
+The response contains two elements:
+
+`messageHistory` is the full history of previous message, from user and assistant, including the latest message written by the assistant
+`newMessage` is the latest message written by the assistant
+
+Ideally, the messages history needs to transit within the process and needs to be the input of this `Converse` task.
+With the new message
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/amazon-bedrock.md:130]: I think this was either carried down a line or an incomplete sentence, but made an adjustment in my latest commit.

**Comment by mathias-vandaele** [comment]
> This will need to be added to the sidebars.js file 👍

I have done it @christinaausley

**Review by sbuettner — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]
I moved it around in the sidebar to accommodate the alphabetical style. Nice work!

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@mathias-vandaele @sbuettner I assume this is ready for release given the review discussion -- merging.

**Pull request merged**